install/build.py

```diff
@@ -164,13 +164,17 @@ def get_compiler_setting(use_hip):
         else:
             define_macros.append(('CUPY_NO_NVTX', '1'))
 
+    # for <cupy/complex.cuh>
+    cupy_header = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                               '../cupy/core/include')
     cub_path = os.environ.get('CUB_PATH', '')
     if os.path.exists(cub_path):
-        # for <cupy/complex.cuh>
-        cupy_header = os.path.join(os.path.dirname(os.path.realpath(__file__)),
-                                   '../cupy/core/include')
         include_dirs.append(cupy_header)
         include_dirs.append(cub_path)
```

Comment:
This part will need to be changed (do prepend/insert instead of append), because `cub_path` would otherwise be searched after `cuda_path`, so for CUDA 11+ we'd still end up with the built-in CUB.

Reply:
That's exactly right. I will fix it. Thanks!